test/e2e/docker/compose.yaml.tmpl

```diff
@@ -44,7 +44,7 @@ services:
       e2e: true
     container_name: {{ .Chain.Name }}
     image: ghcr.io/foundry-rs/foundry:latest
-    entrypoint: ['anvil','--host','0.0.0.0','--chain-id','{{ .Chain.ID }}','--block-time','1', '--silent', {{ if .LoadState }}'--load-state','/anvil/state.json'{{ end }}]
+    entrypoint: ['anvil','--host','0.0.0.0','--chain-id','{{ .Chain.ID }}','--block-time','1', {{ if .LoadState }}'--load-state','/anvil/state.json'{{ end }}]
```

Comment:
no need to make this silent. docker logs are much more helpful

Reply:
the problem is that is spams tons and tons of logs which then make debugging failed e2e tests basically impossible as it overloads the github UI. Best would be to make it verbose when debugging locally but make it silent when running in CI